utils/scarb-test-support/src/command.rs

```diff
@@ -9,37 +12,40 @@ use std::path::{Path, PathBuf};
 use std::process::Command as StdCommand;
 use std::sync::LazyLock;
 
-use crate::cargo::cargo_bin;
-
 pub struct Scarb {
-    cache: EnvPath,
+    cache: Option<EnvPath>,
     config: EnvPath,
```

Comment:
Can we doc that None = shared? Or changed to enum? 🤔

Reply:
fe398a1a58efd1c7266b8f2070f20835c2f949e5